### PR TITLE
docs: add stackblitz links  for some example apps

### PR DESCRIPTION
**Pull request by soulsam480**
Following #1720 , I've added some StackBlitz links for  `bare node`, `expressjs` and `fastify` server examples. There is one issue with the bare node server example on StackBlitz which I'm unable to fix . If anyone can look into that it'd be a great help. Here is the link  https://stackblitz.com/edit/trpc-standalone-server?file=README.md . Thx

### CODE REVIEW

**Comment by sachinraja** [comment]
Nodemon doesn't seem to be working correctly either. The server isn't stopped after a reload, so when Nodemon restarts, it will error because there is already something listening on that port.

File: www/docs/main/example-apps.md [hunk 1]
```diff
@@ -102,6 +103,7 @@ Here's some example apps:
       <td>
         <ul>
           <li><a href="https://githubbox.com/trpc/trpc/tree/main/examples/express-server">CodeSandbox</a></li>
+          <li><a href="https://stackblitz.com/edit/trpc-express-server?file=README.md">StackBlitz</a></li>
```
Comment (KATT) [inline www/docs/main/example-apps.md:106]: This should be something like

https://stackblitz.com/github/trpc/trpc/tree/main/examples/express-server

File: www/docs/main/example-apps.md [hunk 2]
```diff
@@ -92,6 +92,7 @@ Here's some example apps:
       <td>
         <ul>
           <li><a href="https://githubbox.com/trpc/trpc/tree/main/examples/standalone-server">CodeSandbox</a></li>
+          <li><a href="https://stackblitz.com/edit/trpc-standalone-server?file=README.md">StackBlitz</a></li>
```
Comment (KATT) [inline www/docs/main/example-apps.md:95]: See next comment

File: www/docs/main/example-apps.md [hunk 3]
```diff
@@ -112,6 +114,7 @@ Here's some example apps:
       <td>
         <ul>
           <li><a href="https://codesandbox.io/s/github/trpc/trpc/tree/main/examples/fastify-server">CodeSandbox</a></li>
+          <li><a href="https://stackblitz.com/edit/trpc-fastify-server?file=README.md">StackBlitz</a></li>
```
Comment (KATT) [inline www/docs/main/example-apps.md:117]: See previous comment

**Review by KATT — CHANGES_REQUESTED** [review]

**Comment by soulsam480** [comment]
> Nodemon doesn't seem to be working correctly either. The server isn't stopped after a reload, so when Nodemon restarts, it will error because there is already something listening on that port.

Yes it's only that example which is having issues. Can you help ?

File: www/docs/main/example-apps.md (hunk 1 above)
Comment (soulsam480) [reply www/docs/main/example-apps.md:106]: fixing them

**Comment by KATT** [comment]
see #1761 for an example

**Pull request closed without merging**